code/__DEFINES/loadout.dm

```diff
@@ -546,6 +552,6 @@ GLOBAL_LIST_INIT(bypass_storage_content_save, typecacheof(list(
 #define MAXIMUM_LOADOUT 50
 
 /// The current loadout version
-#define CURRENT_LOADOUT_VERSION 6
+#define CURRENT_LOADOUT_VERSION 7
 
 GLOBAL_LIST_INIT(accepted_loadout_versions, list(5,6))
```

Comment:
this has to change too. And delete the previous version transition code i did in save